package/gluon-mesh-vpn-wireguard/files/lib/gluon/mesh-vpn/wireguard_pubkey.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+INCLUDE_ONLY=1
+. /lib/netifd/proto/wireguard.sh
+
+ensure_key_is_generated vpn
+uci get network.vpn.private_key | /usr/bin/wg pubkey
```

Comment:
`/usr/bin/wg` -> `wg`?  Why wouldn't /usr/bin be in PATH here? I assume `uci` is in /usr/bin too.

Reply:
No, it's in `/sbin/uci`.